pandas/core/generic.py

```diff
@@ -5661,22 +5661,59 @@ def clip_lower(self, threshold, axis=None, inplace=False):
         """
         Return copy of the input with values below given value(s) truncated.
 
+        It truncates values below a certain threshold. Threshold can be a single
+        value or an array, in the latter case it performs the truncation
+        element-wise.
+
         Parameters
         ----------
         threshold : float or array_like
+            Minimum value allowed. All values below threshold will be
+            set to this value.
         axis : int or string axis name, optional
             Align object with threshold along the given axis.
         inplace : boolean, default False
             Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+                .. versionadded:: 0.21.0.
```

Comment:
The dot should go in the previous line. If the validation script complaints about it, ignore the message.